group_vars/all/main.yaml

```diff
@@ -1,3 +1,3 @@
-kubevirtRelease: sprint5
+kubevirtRelease: sprint5.1
```

Comment:
once https://travis-ci.org/kubevirt/kubevirt/builds/346890274 is done this can be merged.